scripts/ast-patches/allowAnyPropOn$wEventTarget.ts

```diff
@@ -0,0 +1,31 @@
+import ts from "typescript";
+import {
+  is$wModule,
+  isInterfaceMixinMember,
+  isMixinInterface,
+  Writable
+} from "./utils";
+
+const allowAnyPropOn$wEventTarget = (ast: ts.SourceFile): ts.SourceFile => {
+  const $w = ast.statements.find(is$wModule);
+  if (!$w || !$w.body || !ts.isModuleBlock($w.body)) return ast;
+
+  const $wEvent = $w.body.statements.find(
+    (statement): statement is ts.InterfaceDeclaration =>
+      isMixinInterface(statement, "Event")
+  );
+  if (!$wEvent) return ast;
```

Comment:
Don't we have a prettier rule to prevent this? or eslint?
Please use the curly braces form of `if` because this one can cause actual bugs.
If we don't have a rule, can you add one? It will auto fix the rest of these in your code.